docs/guides/migrating-from-1-to-2.md

```diff
@@ -0,0 +1,9 @@
+## React Dropdown Tree Select Migration Guide v1 to v2
```

Comment:
We shouldn't be needing this for radio select. It's a new addition that's backward compatible.

Reply:
Reverted